backend/README.md

```diff
@@ -20,5 +20,28 @@ Run `stack exec -- edna-server -c dev-config.yaml` or `cabal run -- edna-server
 * The server endpoints will be available at `http://localhost:9000/api/*`.
 * Swagger docs will be available at `http://localhost:9000/docs/`.
 
-Set `EDNA_DEBUG_DB=1` to enable logging of all DB actions made.
+Set `EDNA_DB_DEBUG=1` to enable logging of all DB actions made.
 It works in tests as well.
+
+Also you can pass other command line arguments, to see full list run `stack exec -- edna-server -h` or `cabal run -- edna-server -h`:
+```
+Usage: edna-server [-c|--config EDNA_CONFIG] [--listen-addr EDNA_API_LISTEN]
```

Comment:
I slightly dislike having the full help message in README because we'll most likely forget to update it when we add a new option or change existing ones. Without automated checks it's almost impossible to keep it up-to-date. I think just mentioning `edna-server -h` (as you do) is already sufficient.

Reply:
Why are these conversations marked as resolved if they are not? I mean, there are no requested changes (maybe you forgot to push?). And usually the person who requested changes marks a conversation as resolved after checking that changes have been made properly.